code/game/gamemodes/meteor/meteor.dm

```diff
@@ -69,6 +69,7 @@
 		universe.OnExit()
 
 	var/datum/universal_state/meteor_storm/meteor_storm = new /datum/universal_state/meteor_storm
+	universe = meteor_storm
```

Comment:
I never properly reviewed the universe meteors PR but christ you should know better than this. Fucking fix the code now or this PR isn't getting merged.